docs/config/configuring-nodes-storage.rst

```diff
@@ -51,3 +51,6 @@ The following pages describe how to configure the backends currently supported b
    emc-configuration
    zfs-configuration
    loopback-configuration
+   
+Flocker supports pluggable storage backends. 
```

Comment:
No, this was addressed by @davidoliverSP2 and the comment is now obsolete.

It was something like:

`Sentence one. Sentence two.`

and it should be:

```
Sentence one.
Sentence two.
```

Which renders the same but can be easier to read diffs for. See http://rhodesmill.org/brandon/2012/one-sentence-per-line/.